doc/README.rst

```diff
@@ -0,0 +1,255 @@
+======================================
+
+The new base images and all the images based on them, as the migrated GE images,
+have a support account. There are two public keys with an important role in the
+support account:
+
+- a SSH public key: This is the only method to log in the virtual machine with
+  the support account via SSH (the password access is disabled).
+- a GPG public key: This key is used to encrypt the password of the support
+  account. That password is generated inside the VM and after the encryption
+  is printed to the console. The support account password allows to access the
+  VM via console, which is useful if the network is down. Besides, the password
+  is also needed to get full root privileges after connecting to the support
+  account both using console and ssh client due to for security reasons sudo is
+  configured for asking the password.
+
+Another reason to require the support password and encrypt it is to make easy
+enforcing full control and audit of who have admin access to each VM. This is
+because the SSH public is not specific of each VM, but the password yes. There
+are alternatives but they are more complex. For example, do not provide the SSH
+private key to the support staff but use an special designed ssh-agent that audit
+each access. This document describes the `protocol used by OpenSSH's ssh-agent`_.
+
+Top_
+
+
+Why each region must use their own public keys
+==============================================
+
+The region staff team are responsible of the virtual machines instantiated on their
+servers. Therefore each organisation should have the control of who access the virtual
+machines for support purposes and set and enforce the corresponding policy. It is
+not possible if the public keys are shared among all the organisations. Additionally,
+it is also extremely insecure and a problem when an organisation left the federation.
+
+Top_
+
+
+How the keys are injected inside the virtual machines. Limitations
+==================================================================
+
+The support account is created by a script invoked at boot time. This script is named
+*activate_support_account.py* and it is available at `GlanceSync support scripts`_.
+Inside this folder there is also documentation and all the material to create the base
+images with the support account.
+
+The first thing that the script has to do is to check if a support account is not already
+created and the UUID of the virtual machine has not changed; in these cases the script
+end after printing by console the encrypted password. To encrypt this password we use
+the public GPG key provided by the FIWARE Lab administrator node. To decrypt the password
+you have to use your private GPG key, we have developed an specific script that can be
+found in section `Obtaining the password of the support account`_.
+
+The checking of the UUID is done because a template could be created from a running virtual
```

Comment:
Perhaps only with the link to that README (the documentation of the script in the GlanceSync repo) it is enough (for not repeat the same content), and in this documentation only to leave the key points. Any way, LGTM.